quicktests/overlaying/tests/realistic/vertical_line.js

```diff
@@ -9,20 +9,23 @@ function run(svg, data, Plottable) {
 
   var ds = new Plottable.Dataset(data);
 
+  var customFormatter = function(d){
+    return d + " units";
```

Comment:
Might want to do a rounding or use `toFixed()`/`toPrecision()` here (or base it off an existing `Formatter`:
<img width="314" alt="screen shot 2015-08-31 at 11 26 28 am" src="https://cloud.githubusercontent.com/assets/1440449/9586697/255f2bd2-4fd3-11e5-8e74-16f54a80b8ab.png">

The change in `Axis` size makes things jumpy even though `autorangeSmooth()` is enabled.